schedule pattern

```diff
@@ -0,0 +1,69 @@
+import java.util.ArrayList;  
+import java.util.List;  
+interface Job {  
+    void execute();  
+}  
+interface Trigger {  
+    boolean shouldRun();  
+}  
+class Scheduler {  
+    private List<Job> jobs;  
+    private List<Trigger> triggers;  
+    public Scheduler() {  
+        this.jobs = new ArrayList<>();  
+        this.triggers = new ArrayList<>();  
+    }  
+    public void addJob(Job job, Trigger trigger) {  
+        jobs.add(job);  
+        triggers.add(trigger);  
+    }  
+    public void start() {  
```

Comment:
actually we use spring boot and the framework itself provides centralized system and the dev just needs to use specific stereotype annotations to schedule job with configs